recipes/_install-windows.rb

```diff
@@ -111,4 +139,12 @@
   else
     success_codes [0, 3010]
   end
+  not_if do
+    require 'digest'
+
+    unsafe = unsafe_hashsums.include? Digest::SHA256.file(temp_file).hexdigest
+    Chef::Log.info("\n#{fix_message}\n") if unsafe
```

Comment:
nit: we are not failing the chef run on this, but we are silently not installing the Agent. `warning` could be abetter log level here. Also, an additional line about the fact that we're `continuing without installing Datadog Agent` would be helpful :)

Also, note that the recipes that run after this recipe may rely on the Agent to be installed to work, so on hosts that do not already have the Agent installed subsequent recipes may fail the Chef run. Not avoidable IMO, but it may be useful to be document this on the `windows_blacklist_silent_fail` attribute? Up to you, I don't have a strong opinion on this.

Reply:
indeed, the chef runs might fail after depending on circumstances. Not sure we can do much more than what we're currently doing. The attribute is a best effort approach. Will add a comment to it.